dips/dip-1.mdx

```diff
@@ -11,141 +11,77 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-
-The remainder of this document details the specifications for Commands, Objects and the typical and exceptional flows to define Payments and their Commands.
-
----
 # Specification
 ---
+## On-Chain Account Setup {#on-chain-account-setup}
 
-### Terminology
-
-**Object**: A record. A PaymentObject is an example of an Object representing a payment.
-
-**Command**: An instruction sent over a Channel to mutate/create one or more Objects. In the case of a mutation, this Command will depend upon the current value of one of more existing Shared Objects.
+A VASP’s on-chain account must be created and set up in accordance with the standard VASP account creation process.
 
-**Channel**: The communication path between a pair of entities who execute Commands and track the evolution of a set of Shared Objects.
+Of particular note for off-chain APIs is the compliance key and base url, [DualAttestation::Credential](https://github.com/diem/diem/blob/release-1.1/language/stdlib/modules/DualAttestation.move)’s `compliance_public_key` and `base_url`. The compliance key value is the Ed25519 key with which a VASP signs travel rule statements which are verified on-chain and all off-chain requests and responses. During account setup, a VASP should set their compliance key and base url which will be stored under the Parent VASP account, see [DIP-2](https://github.com/diem/dip/blob/master/dips/dip-2.md) for more information on on-chain roles.
 
-**Shared Object**: All Objects contained within a Command are logically shared, meaning that each VASP has a copy of the Object and may create a new Command to modify it. For example, during the lifecycle of a `KycDataObject`, both VASPs will add information to it. The protocol does not support simultaneous updates to Objects. Rather, VASPs must take turns updating the Object - with turns specified as per the [command sequence](#paymentobject-protocol-command-sequence).
+## HTTP End-point
 
-**Reference ID**: Every Object type contains a `reference_id` field which is a unique reference ID for the Object. The reference ID is always specified by the payment initiator VASP (the VASP which originally created the Object). This value should be globally unique. It is recommended to use a 128 bit long UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122) with "-"'s included.
+Each VASP exposes an HTTPS POST end point at `https://&lt;hostname>:&lt;port>/&lt;protocol_version>/command`. The protocol_version is v2 for this iteration of the off-chain APIs.
```

Comment:
Current version need bump up to "v3"

Reply:
I thought the intent of some of these changes would be to make it be done in a compatible way so that you don't need to bump the version number. If we're bumping the version number then we can and should make a lot more changes.

Reply:
Why are we bumping it to v3? This doesn't contain any breaking changes with the current protocol. Which is why I'm not sure why we included new mandatory fields in the specification doc ;)

Reply:
I think bump up version for the change added.
I feel confusing when doc has v3, but the protocol version is still v2. 

But up to you.